<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Arinc665Qt::DecompileMediaSetWindow</class>
 <widget class="QMainWindow" name="Arinc665Qt::DecompileMediaSetWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/arinc665_qt_resources.qrc">
    <normaloff>:/media_set_manager/arinc665_media_set_decompile.svg</normaloff>:/media_set_manager/arinc665_media_set_decompile.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="Arinc665Qt::Media::MediaSetViewWidget" name="mediaSetView" native="true"/>
      <widget class="QTableView" name="filePathMapping">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDecompileMediaSet"/>
   <addaction name="actionSaveMediaSetXml"/>
  </widget>
  <action name="actionSaveMediaSetXml">
   <property name="icon">
    <iconset theme="document-save"/>
   </property>
   <property name="text">
    <string>Save Media Set XML</string>
   </property>
  </action>
  <action name="actionDecompileMediaSet">
   <property name="icon">
    <iconset theme="document-open"/>
   </property>
   <property name="text">
    <string>Decompile Media Set</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Arinc665Qt::Media::MediaSetViewWidget</class>
   <extends>QWidget</extends>
   <header location="global">arinc665_qt/media/MediaSetViewWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/arinc665_qt_resources.qrc"/>
 </resources>
 <connections/>
</ui>
